docs/admin/Installation/Logging.md

```diff
@@ -1,172 +1,48 @@
-        Read_from_Head      false
-
-    [FILTER]
-        Name record_modifier
-        Match *
-        Record hostname ${HOSTNAME}
-        Record datastoreid ${CLUSTER_UUID}
-
-    [OUTPUT]
-        Name        loki
-        Match       *
-        Host        loki.s9s-dev.net
-        port        443
-        tenant_id   1
-        labels      job="fluent-bit"
-        tls         on
-        tls.verify  off
-        auto_kubernetes_labels on
+* ccxdeps must be updated and deployed.
+```
+helm repo update
+helm upgrade --install ccxdeps s9s/ccxdeps --debug
 ```
 
-> Note: Don't change the input plugin path `[INPUT].Path` and filter record `[FILTER]`.
-
-Input plugins, filters are prepared with the desired settings and are recommended default configuration that should be ready to use without any modifications. If you still need to customize it, you can do that but don't change the `[INPUT].Path` and filter record `[FILTER]`.
-
-> Note: Modifications are needed only on 'output plugins'.
-
-The following configurations are in files/fluent-bit.conf automatically generated if `.Values.fluentbit.enabled` to `true` and `.Values.fluentbit.config` is not defined.
-
-This configuration mounts the fluent-bit.conf configuration file from the ConfigMap into the /etc/fluent-bit/ directory in the host.
+#### Configuring the FQDN of Loki URL
+In our CCX Application, we use loki as log aggregation tool. Customers can choose one of the following options to set the FQDN for the Loki service:
 
-### Example Fluent Bit Configuration to send logs to Elasticsearch output plugin:
+###### Option 1: Change the FQDN of ccxdeps-loki-gateway Ingress
+You can modify the FQDN by updating the ingress host configuration in ccxdeps chart values.yaml file. This ensures that Fluent Bit forwards logs to the correct Loki endpoint.
 
-```yaml
-fluentbit:
+ How to change FQDN: 
+Update the ingress rules in your Helm values file to set the desired FQDN in ccx-monitoring.loki.loki_host_url.
+```
+helm show values s9s/ccxdeps
+```
+```
+ccx-monitoring:
   enabled: true
-  config: |
-    [SERVICE]
-        Flush                     5
-        Log_Level                 info
-        Daemon                    off
-        Parsers_File              parsers.conf
-        HTTP_Server               On
-        HTTP_Listen               0.0.0.0
-        HTTP_Port                 2020
-        storage.path              /var/fluent-bit/state/flb-storage/
-        storage.sync              normal
-        storage.checksum          off
-        storage.backlog.mem_limit 5M
-
-    [INPUT]
-        Name                tail
-        Tag                 ccx.mysql
-        Path                /var/log/mysql/mysqld.log
-        DB                  /var/fluent-bit/state/flb_mysql.db
-        Mem_Buf_Limit       5MB
-        Skip_Long_Lines     On
-        Refresh_Interval    10
-        Read_from_Head      false
+  loki:
+    loki_host_url: &loki_host_url "REPLACE_ME" # Set your custom Loki FQDN here
```

Comment:
why would I need that? where do I get FQDN?

Reply:
it will be automatically set if you didn't specify fqdn. We offer the customer to choose fqdn if they want

Reply:
this is needed to forward the logs to loki instance